service.partymode.autostart/main.py

```diff
@@ -0,0 +1,20 @@
+import xbmc, xbmcaddon
+
+def handleArg(arg):
+
+    pass
+
+def main():
+
+    xbmcaddon.Addon().openSettings()
+
+if __name__ == '__main__':
+
+    args = None
+    if len(sys.argv) > 1:
+        args = sys.argv[1:]
+
+    if args:
```

Comment:
Unused

Reply:
Removed...but for me could be stay there. Maybe, could be useful for some new features